tests/integration/test_imports_no_circular.py

```
@@ -0,0 +1,46 @@
+def test_import_core_and_dsl_without_cycles():
+    # Smoke import test to catch circular-import regressions in core/dsl wiring.
+    from flujo.application.core.executor_core import ExecutorCore  # noqa: F401
```

Ruff (0.14.7)

Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)